store/store.go

```diff
@@ -170,11 +170,18 @@ func (s *Store) createFile(
 	return false, nil
 }
 
-func (s *Store) linkHeight(datahash share.DataHash, height uint64) error {
```

Comment:
What about keeping this method usage as is, but change it to check for empty root and if so make it symlink with proper comment. In other words, decision on whether its symlink or hardlink stays an impl detail of linkHeight